packages/webstudio-sdk/package.json

```
@@ -0,0 +1,77 @@
+{
+  "name": "@webstudio-is/sdk",
+  "version": "0.5.0",
+  "description": "Webstudio JavaScript / TypeScript API",
+  "author": "Webstudio <github@webstudio.is>",
+  "repository": "git@github.com:webstudio-is/webstudio-sdk.git",
+  "homepage": "https://webstudio.is",
+  "scripts": {
+    "build": "yarn build:mdn-data && yarn build:args && yarn build:prisma && yarn build:lib",
+    "build:mdn-data": "node --experimental-loader esbuild-node-loader ./bin/mdn-data.ts ./src/css",
+    "build:args": "node --experimental-loader esbuild-node-loader ./bin/generate-arg-types.ts './src/components/*.tsx !./src/**/*.stories.tsx'",
+    "build:prisma": "prisma format && prisma generate",
+    "typecheck": "tsc --noEmit",
+    "build:lib": "rm -fr lib && tsc",
+    "postinstall": "prisma generate",
+    "test": "jest",
+    "checks": "yarn typecheck && yarn test",
+    "prepublishOnly": "yarn typecheck && yarn build",
```

I would change it to
```suggestion
    "prepublishOnly": "yarn checks && yarn build:lib",
```